godot-kotlin/godot-library/src/nativeMain/kotlin/godot/annotation/PropertyTypeHintAnnotation.kt

```diff
@@ -0,0 +1,246 @@
+package godot.annotation
```

Comment:
Hi ! Great job !
It seems it misses those hint flags:
```
PROPERTY_HINT_LENGTH
PROPERTY_HINT_SPRITE_FRAME
PROPERTY_HINT_KEY_ACCEL
PROPERTY_HINT_IMAGE_COMPRESS_LOSSY
PROPERTY_HINT_IMAGE_COMPRESS_LOSSLESS
PROPERTY_HINT_OBJECT_TOO_BIG
PROPERTY_HINT_INT_IS_OBJECTID
PROPERTY_HINT_MAX
```
Property hint is implemented [here](https://github.com/godotengine/godot/blob/3.2/core/object.h).
Also shouldn't we have hints in same order than in the engine, it would be easier to maintain.

Reply:
Dammit... Completely overlooked that file. That would have made it soo much easier ^^
Thanks I'll maybe implement them (depending on the outcome of the discussion below.

Reply:
I hate it how even in code their documentation is either wrong or outdated if present at all...
`PROPERTY_HINT_RANGE, ///< hint_text = "min,max,step,slider; //slider is optional"`
there is no slider implemented, at the last param it either accepts `or_greater` or `or_lesser` -.-

Reply:
FYI: LENGHT is deprecated: https://github.com/godotengine/godot/blob/e2b66cacf78ae39b94df748e9740b98a1f011e77/doc/classes/%40GlobalScope.xml

Reply:
`PROPERTY_HINT_SPRITE_FRAME` and `PROPERTY_HINT_KEY_ACCEL` as well